### PR TITLE
A script to create an Etherpad release

### PR DESCRIPTION
**Pull request by JohnMcLear**
cc @gared

### CODE REVIEW

File: bin/release.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+// run npm version `release` where release is patch, minor or major
+child_process.execSync('npm install --package-lock-only', {cwd: `src/`});
+// run npm install --package-lock-only <-- required???
+
+child_process.execSync(`git checkout -b release/${newVersion}`);
+child_process.execSync(`git push origin release/${newVersion}`);
+
+child_process.execSync(`make docs`);
+child_process.execSync(`git@github.com:ether/ether.github.com.git`);
```
Comment (Gared) [inline bin/release.js:53]: Do you want a git clone here?

File: bin/release.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+fs.writeFileSync('src/package.json', JSON.stringify(packageJson, null, 2));
+
+// run npm version `release` where release is patch, minor or major
+child_process.execSync('npm install --package-lock-only', {cwd: `src/`});
+// run npm install --package-lock-only <-- required???
+
+child_process.execSync(`git checkout -b release/${newVersion}`);
+child_process.execSync(`git push origin release/${newVersion}`);
+
+child_process.execSync(`make docs`);
```
Comment (Gared) [inline bin/release.js:52]: Maybe also generate the windows-build here?

**Review by Gared — APPROVED** [review]

File: bin/release.js (hunk 2 above)
Comment (JohnMcLear) [reply bin/release.js:52]: I thought the windows build auto generated by a script you made?

File: bin/release.js (hunk 1 above)
Comment (JohnMcLear) [reply bin/release.js:53]: Good spot and yes!

**Review by AKJK-Internet-32002 — APPROVED** [review]

**Review by AKJK-Internet-32002 — COMMENTED** [review]
can I use this on my termux or matrix app

File: bin/release.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+child_process.execSync(`cit clone git@github.com:ether/ether.github.com.git`);
```
Comment (alangecker) [inline bin/release.js:53]: `cit`, should possibly be `git`? :)

Comment (JohnMcLear) [reply bin/release.js:53]: lol yep 👍

**Pull request merged**